src/helpers.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+if (function_exists('horizon_stats')) {
```

Comment:
This is wrong. The correct here is to check if this function was not declared, so you need to use a `!` here.